utils/_features.py

```diff
@@ -1828,3 +1828,39 @@ def appsec_user_blocking(test_object):
         """
         pytest.mark.features(feature_id=254)(test_object)
         return test_object
+
+    @staticmethod
+    def kafkaspan_creationcontext_propagation_with_dd_trace_js(test_object):
+        """
+        [Kafka][Span Creation][Context Propagation] with dd-trace-js
+        https://feature-parity.us1.prod.dog/#/?feature=255
+        """
+        pytest.mark.features(feature_id=255)(test_object)
```

Comment:
Should these all be the same feature? (Legit question)

Reply:
No, we want one feature per language

Reply:
Yes, one test class tests one "buddy", and it's one feature. And we have 7 tests classes, one per buddy.

Each buddy can simply seen as yet another component to interact with, like a DB for instance. We would have called the test class `Test_SomeDbEngineIntegration`.

Just, it turns out this component is a DD tracer + kafka.